app/controllers/guests_controller.rb

```diff
@@ -0,0 +1,17 @@
+class GuestsController < ApplicationController 
+
+  def show 
+    @guest = Guest.find(params[:id])
+    @guest_rooms = Guest.find(params[:id]).rooms.first
+  end
```

Comment:
On line 5, you are choosing the first room off of the collection, but don't we want to show all of the rooms for that guest? 

I think we'll only need to have line 4 in this method. In the view, we can call`@guest.rooms` in order to get the entire collection of rooms that's associated with that guest.